src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,62 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ITEMS_NUMBER = 10;
+    private static final int MAX_WIDTH_NUMBER = 2;
+    private Object [][] storage;
```

Comment:
```suggestion
    private K[] keys;
    private V[] values;
```

With such approach your solution would be easier to read